### DISCUSSION TITLE
What is the best way to converse with my crew and provide feedback.

### DISCUSSION THREAD

**Discussion by madhavthaker1** [crewAIInc/crewAI#1245] [Q&A]
Let's say my crew is generating ad copy and I want to make slight suggestions. How can I continue the session with my crew while giving it feedback. Essentially, i'm looking for a `UserProxyAgent` feature that exists in Autogen.

**Comment by theCyberTech** [comment]
Maybe this will help - https://docs.crewai.com/how-to/Human-Input-on-Execution/

**Comment by madhavthaker1** [comment]
This is great, thanks. This seems to the allow for one round of feedback. Curious if there are ways to allow for many rounds of feedback until I've said the result is good enough.

**Comment by theCyberTech** [comment]
You could probably do it with some Python magic, but Im not sure. In the mean time Ill tag this as a feature request
